bikespace_frontend/src/pages/index.tsx

```diff
@@ -4,50 +4,74 @@ import { button, title, logo, main, mainContent } from "../styles/index.css";
 import { StaticImage } from "gatsby-plugin-image";
 
 type DataProps = {
-    site: {
-        siteMetadata: {
-            title: string;
-        };
+  site: {
+    siteMetadata: {
+      title: string;
     };
+  };
 };
 
 const IndexRoute = ({ data: { site } }: PageProps<DataProps>) => {
-    return (
-        <main className={main}>
-            <div className={mainContent}>
-                <h1 className={title}>{site.siteMetadata.title}</h1>
-                <StaticImage
-                    className={logo}
-                    src="../images/bikespace-logo.svg"
-                    alt="BikeSpace Logo"
+  return (
+    <main className={main}>
+      <header>
+        <nav className={"main-nav"} aria-label="Main">
+          <ul>
+            <li>
+              <a href="https://bikespace.ca/">About BikeSpace</a>
+            </li>
+            <li>
+              <a href="https://dashboard.bikespace.ca/">Explore the Data</a>
+            </li>
+            <li>
+              <a href="https://github.com/bikespace/bikespace/tree/main/bikespace_frontend">
```

Comment:
Would it be better to navigate to https://github.com/bikespace/bikespace ?
The frontend folder's README is just the boilerplate gatsby one.

Reply:
Should we move that README here then and write a new one for the root folder just outlining the different sub folders? (App, api, dashboard, landing page) happy to draft something for the root if you think this makes sense

Reply:
Yeah it's on my list for #78 to cleanup the READMEs. I'll update/add READMEs for the missing ones like frontend, api and landing page.